packages/console/src/components/Markdown/index.tsx

```diff
@@ -12,29 +12,73 @@ type Props = {
   children: string;
 };
 
-const Markdown = ({ className, children }: Props) => (
-  <ReactMarkdown
-    remarkPlugins={[remarkGfm]}
-    className={classNames(styles.markdown, className)}
-    components={{
-      code: ({ node, inline, className, children, ...props }) => {
-        const [, codeBlockType] = /language-(\w+)/.exec(className ?? '') ?? [];
-
-        return inline ? (
-          <code className={styles.inlineCode} {...props}>
-            {children}
-          </code>
-        ) : (
-          <CodeEditor isReadonly language={codeBlockType} value={String(children)} />
+const Markdown = ({ className, children }: Props) => {
+  const tocIdList = React.useRef<string[]>([]);
+
+  const generateTocId = (text: string): string | undefined => {
+    const initialKebabCaseString = text.replace(/\s/g, '-').toLowerCase();
+
+    const checkIdCollision = (kebabCaseString: string, index = 0): string | undefined => {
+      if (!tocIdList.current.includes(kebabCaseString)) {
```

Comment:
use map？

Reply:
A simple list would do the trick, as we only need to store the generated id, so I don't think we need a key value map